.github/workflows/process-fbc-fragment.yaml

```
@@ -0,0 +1,72 @@
+              PREV_PATCH=0
+              PREV_MINOR=$((MINOR - 1))
+              PREV_VERSION="$MAJOR.$PREV_MINOR.$PREV_PATCH"
+            fi
+          else
+            PREV_PATCH=$((PATCH - 1))
+            PREV_VERSION="$MAJOR.$MINOR.$PREV_PATCH"
+          fi
+          make catalog-prepare -e BUNDLE_IMGS=${QUAY_IMG}:v${PREV_VERSION},${QUAY_IMG}:v${CURRENT_VERSION}
+
+      - name: Push latest FBC artifacts
+        if: ${{ steps.check-if-pcc-cache-valid.outputs.PCC_CACHE_VALID == 'NO' }}
```

actionlint (1.7.7)

property "check-if-pcc-cache-valid" is not defined in object type {generate-fbc-fragment: {conclusion: string; outcome: string; outputs: {string => string}}; get_branch: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)